Makefile

```diff
@@ -100,3 +100,8 @@ lint: lint-go lint-python
 .PHONY: mod-tidy
 mod-tidy:
 	$(GO) mod tidy
+
+.PHONY: vendor-python
+vendor-python:
+	pip install vendoring
```

Comment:
1. Maybe this could be a script in `script/` as we're slowly trying to deprecate the `Makefile`, I think.

2. If you do want it to stay in here then please replace `pip` with `$(PYTHON) -m pip` and `vendoring` with `$(PYTHON) -m vendoring` if possible, otherwise the Makefile breaks when a custom `PYTHON` is supplied.